src/script/E2EIdentity/E2EIdentity.test.ts

```diff
@@ -48,7 +48,7 @@ jest.mock('./Modals', () => ({
     LOADING: 'loading',
     SUCCESS: 'success',
     ERROR: 'error',
-    ENROLL: 'enroll',
+    enrol: 'enrol',
```

Comment:
```suggestion
    ENROL: 'enrol',
```
I think You change all usages :D

Reply:
yep, I fixed it just now :)